src/app/CommandSender.h

```diff
@@ -53,6 +53,8 @@ class CommandSender : public Command, public Messaging::ExchangeDelegate
 
 private:
     CHIP_ERROR ProcessCommandDataElement(CommandDataElement::Parser & aCommandElement) override;
+
+    CommandId mCommandIndex;
```

Comment:
This does not look like the right type, and don't we need to initialize it to 0 somewhere?

Reply:
just use unint8_t, yes, just initialize.